src/client/v2/algod/simulateTransaction.ts

```diff
@@ -59,16 +59,16 @@ export default class SimulateRawTransactions extends JSONRequest<
     const res = await this.c.post(
       this.path(),
       Buffer.from(this.txnBytesToPost),
-      txHeaders
+      txHeaders,
+      this.query,
+      false
```

Comment:
I think the underlying problem is that [`post`](https://github.com/algorand/js-algorand-sdk/blob/0039284f1ecf77b609acab9da3df1e62679f8531/src/client/client.ts#L256) and `delete` assume the response is always `application/json`. Probably they should use `getAcceptFormat(query)` like `get` does to figure out the actual response type.

But I'm ok with leaving this as-is for now, since passing `parseBody` as false is sufficient for the time being

Reply:
I opened https://github.com/algorand/js-algorand-sdk/issues/751